src/main/java/com/atlauncher/gui/tabs/VanillaPacksTab.java

```diff
@@ -233,177 +180,69 @@ private void checkDirty(DocumentEvent e) {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                if (minecraftVersionBetasFilterCheckbox.isSelected()) {
-                    minecraftVersionTypeFilters.add(VersionManifestVersionType.OLD_BETA);
-                } else {
-                    minecraftVersionTypeFilters.remove(VersionManifestVersionType.OLD_BETA);
-                }
-
-                reloadMinecraftVersionsTable();
-            }
-        });
-        if (ConfigManager.getConfigItem("minecraft.old_alpha.enabled", true) == true) {
-            minecraftVersionFilterPanel.add(minecraftVersionBetasFilterCheckbox);
-        }
-
-        minecraftVersionAlphasFilterCheckbox.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                if (minecraftVersionAlphasFilterCheckbox.isSelected()) {
-                    minecraftVersionTypeFilters.add(VersionManifestVersionType.OLD_ALPHA);
-                } else {
-                    minecraftVersionTypeFilters.remove(VersionManifestVersionType.OLD_ALPHA);
-                }
-
-                reloadMinecraftVersionsTable();
-            }
-        });
-        if (ConfigManager.getConfigItem("minecraft.old_beta.enabled", true) == true) {
-            minecraftVersionFilterPanel.add(minecraftVersionAlphasFilterCheckbox);
-        }
+        // Old Alphas checkbox
+        setupOldAlphasCheckbox(minecraftVersionFilterPanel);
 
         minecraftVersionPanel.add(minecraftVersionFilterPanel);
-
         mainPanel.add(minecraftVersionPanel, gbc);
-
         gbc.gridx++;
         gbc.insets = UIConstants.FIELD_INSETS;
         gbc.anchor = GridBagConstraints.BASELINE_LEADING;
-
-        JScrollPane minecraftVersionScrollPane = new JScrollPane(JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED,
-                JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
-
+        JScrollPane minecraftVersionScrollPane = new JScrollPane(
+            JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER
+        );
         minecraftVersionScrollPane.setPreferredSize(new Dimension(450, 300));
-
         setupMinecraftVersionsTable();
-        reloadMinecraftVersionsTable();
         minecraftVersionScrollPane.setViewportView(minecraftVersionTable);
-
         mainPanel.add(minecraftVersionScrollPane, gbc);
 
         // Loader Type
         gbc.gridx = 0;
         gbc.gridy++;
         gbc.insets = UIConstants.LABEL_INSETS;
         gbc.anchor = GridBagConstraints.EAST;
-
         JLabel loaderTypeLabel = new JLabel(GetText.tr("Loader") + "?");
         mainPanel.add(loaderTypeLabel, gbc);
-
         gbc.gridx++;
         gbc.insets = UIConstants.FIELD_INSETS;
         gbc.anchor = GridBagConstraints.BASELINE_LEADING;
-
         loaderTypeButtonGroup.add(loaderTypeNoneRadioButton);
+        loaderTypeButtonGroup.add(loaderTypeQuiltRadioButton);
```

Comment:
Move quilt back to the bottom to keep it alphabetical

Reply:
I was following the probable pattern I saw.

`New->Old`

But oki doki

Reply:
[4f7708b](https://github.com/ATLauncher/ATLauncher/pull/743/commits/4f7708bd2111655bf60a9efac7fb986687a8dcd5)